pkg/kubectl/scale.go

```diff
@@ -180,11 +197,63 @@ func (scaler *ReplicationControllerScaler) Scale(namespace, name string, newSize
 	return nil
 }
 
-// ScalerClient abstracts access to ReplicationControllers.
+// ScaleSimple is responsible for updating job's parallelism.
+func (scaler *JobScaler) ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (string, error) {
+	job, err := scaler.c.GetJob(namespace, name)
+	if err != nil {
+		return "", ControllerScaleError{ControllerScaleGetFailure, "Unknown", err}
+	}
+	if preconditions != nil {
+		if err := preconditions.ValidateJob(job); err != nil {
+			return "", err
+		}
+	}
+	parallelism := int(newSize)
```

Comment:
Should parallelism have been a `uint`?  There's no godoc explaining what a negative value would mean.